app/controllers/facilities_controller.rb

```diff
@@ -1,6 +1,35 @@
 class FacilitiesController < ApplicationController
 
   def index
+    #check various types of health facilities
```

Comment:
@nthezemu again, use Ruby Collection Methods to query collections (instead of populating them into empty variables as you are doing), i.e.:

`@hospitals = Facility.all.count { |f| f.type_code == 1 }` will give you the total hospital counts